modules/nix-darwin/templates/default.nix

```diff
@@ -0,0 +1,121 @@
+{ config, pkgs, lib, options, ... }:
+with lib;
+with lib.types;
+with builtins;
+let
+  cfg = config.sops;
+  secretsForUsers = lib.filterAttrs (_: v: v.neededForUsers) cfg.secrets;
+  users = config.users.users;
+  renderScript = ''
+    echo Setting up sops templates...
+    ${concatMapStringsSep "\n" (name:
+      let
+        tpl = config.sops.templates.${name};
+        substitute = pkgs.writers.writePython3 "substitute" { }
+          (readFile ./subs.py);
+        subst-pairs = pkgs.writeText "pairs" (concatMapStringsSep "\n"
+          (name:
+            "${toString config.sops.placeholder.${name}} ${
+              config.sops.secrets.${name}.path
+            }") (attrNames config.sops.secrets));
+      in ''
+        mkdir -p "${dirOf tpl.path}"
+        (umask 077; ${substitute} ${tpl.file} ${subst-pairs} > ${tpl.path})
+        chmod "${tpl.mode}" "${tpl.path}"
+        chown "${tpl.owner}:${tpl.group}" "${tpl.path}"
+      '') (attrNames config.sops.templates)}
+  '';
```

Comment:
We can remove the python script now that templates have been added to the go code: https://github.com/Mic92/sops-nix/pull/649